src/Bridges/Constraint/zero_one.jl

```diff
@@ -0,0 +1,96 @@
+"""
+    ZeroOneBridge{T}
+
+The `ZeroOneBridge` splits a `MOI.SingleVariable`-in-`MOI.ZeroOne` constraint
+into a `MOI.SingleVariable`-in-`MOI.Integer` constraint
+and a `MOI.SingleVariable`-in-`MOI.Interval(0,1)` constraint.
+"""
+struct ZeroOneBridge{T} <: AbstractBridge
+    variable_index::MOI.VariableIndex
+    interval_index::MOI.ConstraintIndex{MOI.SingleVariable, MOI.Interval{T}}
+    integer_index::MOI.ConstraintIndex{MOI.SingleVariable, MOI.Integer}
+end
+
+function bridge_constraint(::Type{ZeroOneBridge{T}}, model::MOI.ModelLike, f::MOI.SingleVariable, ::MOI.ZeroOne) where {T <: Real}
+    interval_index = MOI.add_constraint(model, f, MOI.Interval{T}(zero(T), one(T)))
+    integer_index = MOI.add_constraint(model, f, MOI.Integer())
+    return ZeroOneBridge{T}(f.variable, interval_index, integer_index)
+end
+
+function MOIB.added_constraint_types(::Type{<:ZeroOneBridge{T}}) where {T}
+    return [
+            (MOI.SingleVariable, MOI.Interval{T}),
+            (MOI.SingleVariable, MOI.Integer),
+            ]
+end
+
+function concrete_bridge_type(::Type{<:ZeroOneBridge{T}},
+                              ::Type{MOI.SingleVariable},
+                              ::Type{MOI.ZeroOne}) where {T}
+    return ZeroOneBridge{T}
+end
+
+function MOI.supports_constraint(::Type{<:ZeroOneBridge},
+                                 ::Type{MOI.SingleVariable},
+                                 ::Type{MOI.ZeroOne})
+    return true
+end
+
+
+# Attributes, Bridge acting as a constraint
+function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintSet, bridge::ZeroOneBridge)
+    return MOI.ZeroOne()
+end
+
+function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintFunction, bridge::ZeroOneBridge)
+    return MOI.SingleVariable(bridge.variable_index)
```

Comment:
Do `MOI.get(model, attr, bridge.integer_index)` (or `interval_index`, the choice is yours :) )